dashboard/test/integration/pardot_test.rb

```diff
@@ -0,0 +1,87 @@
+require 'test_helper'
+require 'cdo/contact_rollups'
+
+PEGASUS_TEST_DB_NAME = "pegasus_#{Rails.env}"
+COLUMN_NAME_TO_INDEX_MAP = { "roles": 0, "ages_taught": 1 }.freeze
+
+class PardotTest < ActiveSupport::TestCase
+  def test_empty_contacts
+    # Test the rollup process with an empty database
+    rollups_test_helper 0, []
+  end
+
+  def test_teachers
+    # Create teacher 1 with no section
+    @teacher1 = create(:teacher, email: "rolluptestteacher1@code.org")
+
+    # Create teacher 2 with one section and one student
+    @teacher2 = create(:teacher, email: "rolluptestteacher2@code.org")
+    create_sections_helper @teacher2, [[{age: 6}]]
+
+    # Create teacher 3 with one section and multiple students
+    @teacher3 = create(:teacher, email: "rolluptestteacher3@code.org")
+    @teacher3_section = create(:section, user: @teacher3)
+    create_sections_helper @teacher3, [[{age: 6}, {age: 10}, {age: 14}, {age: 10}]]
```

Comment:
That would be a useful test, and it would make more sense to have a separate teacher with an empty section.